templates/reviewer/review/step3.tpl

```diff
@@ -37,10 +37,10 @@
 		{if $reviewForm}
 			{include file="reviewer/review/reviewFormResponse.tpl"}
 		{else}
-			{fbvFormSection}
+			{fbvFormSection label="submission.comments.canShareWithAuthor"}
 				{fbvElement type="textarea" id="comments" name="comments" value=$comments readonly=$reviewIsClosed label="submission.comments.canShareWithAuthor" rich=true}
 			{/fbvFormSection}
-			{fbvFormSection}
+			{fbvFormSection label="submission.comments.cannotShareWithAuthor"}
 				{fbvElement type="textarea" id="commentsPrivate" name="commentsPrivate" value=$commentsPrivate readonly=$reviewIsClosed label="submission.comments.cannotShareWithAuthor" rich=true}
```

Comment:
I see that these labels are duplicated in the code. Does the `label` appear duplicated anywhere?

Reply:
I think someone already pointed somewhere that the current small label in the bottom of the textarea is not user friendly. So should I remove the old smaller labels? I think the larger one's looked fine.

Reply:
Yes, the section labels are what we are standardizing for the old forms system.

Reply:
check